libraries/aws_ec2_instance.rb

```diff
@@ -1,7 +1,7 @@
 # author: Christoph Hartmann
 
-class Ec2 < Inspec.resource(1)
-  name 'aws_ec2'
+class AwsEc2Instance < Inspec.resource(1)
+  name 'aws_ec2_instance'
   desc 'Verifies settings for an EC2 instance'
 
   example "
```

Comment:
I like the rename, especially the deprecation. (Neat!)

I think the resource name should also be updated in the resource's examples, as well as in the README.

Reply:
Thank you, I am going to improve that one.